### PR TITLE
Update log to 0.4 and bump version

### PR DESCRIPTION
**Pull request by Eijebong**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ name = "bindgen"
 readme = "README.md"
 repository = "https://github.com/rust-lang-nursery/rust-bindgen"
 documentation = "https://docs.rs/bindgen"
-version = "0.32.3"
+version = "0.32.4"
```
Comment (emilio) [inline Cargo.toml]: This version bump needs to be major, since breaking changes have landed since the last release.

Comment (Eijebong) [reply Cargo.toml]: Oh, sorry, I didn't check that ^^

**Comment by Eijebong** [comment]
Failure is legit, I'll fix it later today

**Comment by emilio** [comment]
Looks fine to me, thanks!

@bors-servo r+

**Comment by emilio** [comment]
Failures look flaky.

**Pull request merged**